deployer/templates/heapster.yaml

```diff
@@ -66,7 +66,6 @@ objects:
           - "--source=kubernetes:${MASTER_URL}?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250"
           - "--sink=hawkular:https://hawkular-metrics:443?tenant=_system&labelToTenant=pod_namespace&caCert=/hawkular-cert/hawkular-metrics-ca.certificate&user=%username%&pass=%password%&filter=label(container_name:^/system.slice.*|^/user.slice)"
           - "--logtostderr=true"
-          - "--vmodule=*=4"
```

Comment:
I think the right way to do this would be to specify at the deployer that we want extra logging levels for all the components. I have created an issue to track that: https://github.com/openshift/origin-metrics/issues/25

I think the default should be to not have verbose logging enabled, which this fixes.